src/components/dev-dot-content/mdx-components/mdx-heading-permalink/mdx-heading-permalink.module.css

```diff
@@ -28,7 +28,7 @@
 		opacity: 1;
 		background: var(--token-color-surface-faint);
 
-		& svg {
+		& .icon {
```

Comment:
Unrelated to current PR (from my most previous) but threw it in here to better follow our [CSS Modules best practices](https://www.notion.so/hashicorp/CSS-Modules-753f2647f7aa4a598e28990d9f0d05a8#de383178becc4320b9e358f0982a6ecc)